### PR TITLE
fix(tabs): keyboard accessibility

### LINKED ISSUE #254: Tabs Component keyboard accessibility (AVT 2)
Currently the Up/down arrows change the focused tab, only the left/right arrows should change the focused tab.

> Is this a feature request (new component, new icon), a bug, or a general issue?

bug

> Is this issue related to a specific component?

Tabs component

> What did you expect to happen? What happened instead? What would you like to see changed?

Up/down arrows should not be binded to any Tabs action

> What browser are you working in?

FF 63

> What version of the Carbon Design System are you using?

"carbon-components": "9.32.1",
"carbon-components-angular": "1.9.1"

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IWR

## Steps to reproduce the issue

1. Go to Tabs component [example](https://angular.carbondesignsystem.com/?knob-size=md&knob-showSelectionColumn=true&knob-striped=true&knob-sortable=true&selectedKind=Tabs&selectedStory=Basic&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)
2. Tab into the tab component.
3. Press up or down arrow. Notice that the focused tab changes.

### PR DESCRIPTION
**Pull request by youda97**
Closes #254

### CODE REVIEW

File: src/tabs/tab-headers.component.ts [hunk 1]
```diff
@@ -135,7 +135,7 @@ export class TabHeaders implements AfterViewInit, AfterContentInit {
 	 */
 	@HostListener("keydown", ["$event"])
 	keyboardInput(event) {
-		if (event.key === "ArrowRight" || event.key === "ArrowDown") {
+		if (event.key === "Right" || event.key === "ArrowRight") {
```
Comment (cal-smith) [inline src/tabs/tab-headers.component.ts]: I was under the impression that ArrowUp/ArrowDown were also valid ... Also which browser sends just Right? That may need a comment

Comment (cal-smith) [reply src/tabs/tab-headers.component.ts]: Ah Interesting ... I'd still add a comment for the IE11-ness of "Right", since it's nonstandard

File: src/tabs/tabs.component.ts [hunk 2]
```diff
@@ -65,6 +66,12 @@ export class Tabs implements AfterContentInit {
 	 * @memberof Tabs
 	 */
 	@Input() cacheActive = false;
+	/**
+	 * Set to 'true' to have tabs automatically activated and have their content displayed when they recieve focus.
+	 * @memberof Tabs
+	 */
+	@Input() automaticActivation = false;
```
Comment (cal-smith) [inline src/tabs/tabs.component.ts]: `automaticActivation` is a bit wordy ... either `activation: "auto" | "manual"` or `followFocus: boolean` may be better ... but that's kinda nit-picky.

for #254 it should default to `true`/`"auto"` as well

**Comment by zvonimirfras** [comment]
Is this ready?

**Comment by youda97** [comment]
Yup :)

File: src/tabs/tab-headers.component.ts [hunk 3]
```diff
@@ -135,25 +141,61 @@ export class TabHeaders implements AfterViewInit, AfterContentInit {
...
+
+		if (event.key === "End") {
+			event.preventDefault();
+			if (this.followFocus) {
+				this.selectTab(event.target, tabsArray[this.allTabHeaders.length - 1], this.allTabHeaders.length);
+			}
+			this.allTabHeaders[this.allTabHeaders.length - 1].focus();
+		}
+
+		if (event.key === " " && !this.followFocus) {
```
Comment (zvonimirfras) [inline src/tabs/tab-headers.component.ts]:
```suggestion
		// 	`"Spacebar"` is IE11 specific value
		if ((event.key === " " || event.key === "Spacebar") && !this.followFocus) {
```
